src/crimson/os/seastore/segment_manager/block.cc

```diff
@@ -46,6 +46,38 @@ static write_ertr::future<> do_write(
   });
 }
 
+static write_ertr::future<> do_writev(
+  seastar::file &device,
+  uint64_t offset,
+  bufferlist &bl)
+{
+  logger().debug(
+    "block: do_writev offset {} len {}",
+    offset,
+    bl.length());
+  std::vector<iovec> iov;
+  iov.reserve(bl.get_num_buffers());
+  for (auto& bp : bl.mut_buffers()) {
+    logger().debug("block: do_writev iov length: {}", bp.length());
+    iov.emplace_back(iovec{bp.c_str(), bp.length()});
+  }
```

Comment:
nit, we can reuse `bl.prepare_iov(&iov)`;

Reply:
done:-)

Reply:
Reverted somehow.

Reply:
Oh, sorry